library/Ivoz/Provider/Domain/Model/Company/Company.php

```diff
@@ -21,21 +22,53 @@ class Company extends CompanyAbstract implements CompanyInterface
 
     use CompanyTrait;
 
+    /**
+     * @codeCoverageIgnore
+     * @return array
+     */
     public function getChangeSet()
     {
         return parent::getChangeSet();
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    public function setName($name)
+    {
+        return parent::setName(trim($name));
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getDomainUsers()
+    {
+        $value = parent::getDomainUsers();
+        return trim($value);
+    }
```

Comment:
If we trim the setters it wouldn't be required to trim the getters. I think we should choose (I personally prefer setters tho).

Reply:
I agree